cronjobs.yml

```diff
@@ -9,12 +9,40 @@ cronJobs:
   - name: process-test-mappings
     schedule: "0 1 * * *"
     command: ["/bin/sh", "cronjobs/process_test_mapping_work_items.sh"]
+    resources: # Burstable
+      requests: # guaranteed amount of resources
+        cpu: 500m
+        memory: 6Gi
+      limits: # maximum allowed resources
+        cpu: 2
+        memory: 12Gi
   - name: process-task-mappings
     schedule: "0 2 * * *"
     command: ["/bin/sh", "cronjobs/process_task_mapping_work_items.sh"]
+    resources: # Burstable
+      requests: # guaranteed amount of resources
+        cpu: 500m
+        memory: 6Gi
+      limits: # maximum allowed resources
+        cpu: 2
+        memory: 12Gi
   - name: update-test-mappings
     schedule: "0 3 * * *"
     command: ["/bin/sh", "cronjobs/update_test_mappings.sh"]
+    resources: # Burstable
+      requests: # guaranteed amount of resources
+        cpu: 500m
+        memory: 6Gi
+      limits: # maximum allowed resources
+        cpu: 2
+        memory: 12Gi
   - name: update-task-mappings
     schedule: "0 4 * * *"
     command: ["/bin/sh", "cronjobs/update_task_mappings.sh"]
+    resources: # Burstable
+      requests: # guaranteed amount of resources
+        cpu: 500m
+        memory: 6Gi
+      limits: # maximum allowed resources
+        cpu: 2
+        memory: 12Gi
```

Comment:
Same here, the update cronjobs seem to be staying below 1 CPU and 3 Gi. The process jobs aren't even showing up due to being very short-lived.

Reply:
The normal case is when they run an update. That is a delta so they tend to be small. 

If it needs to be run for the last 6 months then the requirements are much greater. I can lower them and we can see if we hit the limits.